[ADD] Sales - Custom email signatures (#18238)
odoo/documentation: content/applications/sales/sales/sales_quotations/custom_signatures.rst

=======================
Custom email signatures
=======================

Some business workflows require email template signatures to be updated so outgoing communications
display the appropriate contact information for customer follow-up and correspondence.

This is particularly important when responsibility for a customer opportunity is shared between team
members. In these cases, the employee responsible for managing quotes and customer communications
may differ from the employee associated with the sales order for commission purposes.

> Email templates default to the signature of the employee assigned to the record, therefore the

Reviewer: ```suggestion
Email templates default to the signature of the employee assigned to the record; therefore the
```

template should be modified to display the contact information of the employee responsible for
customer follow-up. This ensures customers know who to contact with questions, approvals, and other
correspondence.

.. example::
   In this use-case, the `Sales: Order Confirmation` email template is used to illustrate this

>    concept.

Reviewer: Also, the "in this use case" phrasing makes it sound like the sales order confirmation template is the only place we apply this email template configuration config, when we're really using it as a consistent example throughout the article to teach the reader the skill (: 

How about this slight rewording?

```suggestion
.. note::
   The `Sales: Order Confirmation` email template is used as the example throughout this article to demonstrate how email signatures can be customized.
```

minor comments about the changes:
- since we're not describing the example in this, maybe we use the note admonition instead?
- use-case --> use case because i think i've typically only seen it spelled "use case"

Author: I like it! It is something that can be geeral, but I was focused on this specific use since it seeme dthis was requested the most - but I agree, general is better.

Open the email template
=======================

To modify an email template, first :ref:`activate developer mode <developer-mode/activation>`, then
navigate to :menuselection:`Settings app --> Technical --> Email Templates`. Click on the email
template being modified, in this case, the `Sales: Order Confirmation` template.

Update from field
=================

> On the email template, click into the *Email Configuration* tab. Delete the default text in the

Reviewer: ```suggestion
On the email template, open the *Email Configuration* tab. Delete the default text in the
```
optional

:guilabel:`From` field and replace it with: `{{ (user.email_formatted) }}`.

.. image:: custom_signatures/from-field.png
   :alt: The from field updated with the correct text.

Modify the code
===============

> Click into the *Content* tab, and highlight some text so the toolbar is visible. Click the

Reviewer: ```suggestion
Click the *Content* tab, and highlight some text so the toolbar is visible. Click the
```

optional

:guilabel:`</> (Code view)` button and the text changes to code.

> Search the text for `signature` to find the two instances of the code `object.user_id.signature`.

Reviewer: ```suggestion
Search the code for `signature` to find the two instances of the code `object.user_id.signature`.
```
only if it makes sense?
